ramls/holdings-records-source.raml

```diff
@@ -0,0 +1,47 @@
+#%RAML 1.0
+title: Holdings Records Sources API
+version: v1.0
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost
+
+documentation:
+  - title: Holdings Records Sources API
+    content: This documents the API calls that can be made to query and manage holdings records sources
+
+types:
+  holdingsRecordsSource: !include holdingsrecordssource.json
+  holdingsRecordsSources: !include holdingsrecordssources.json
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  pageable: !include raml-util/traits/pageable.raml
+  searchable: !include raml-util/traits/searchable.raml
+  language: !include raml-util/traits/language.raml
+  validate: !include raml-util/traits/validation.raml
+
+resourceTypes:
+  collection: !include raml-util/rtypes/collection.raml
+  collection-item: !include raml-util/rtypes/item-collection.raml
+
+/holdings-records-sources:
```

Comment:
This path does not match with the path in the module descriptor /holdings-records-source, which suggests it won't work when proxied via Okapi

Personally, I would be tempted to go with `/holdings-sources` or `/holdings-record-sources` rather than records, as I think the focus is on the source rather than the holdings record. What do you think?